app/controllers/api/services_controller.rb

```diff
@@ -65,8 +64,19 @@ def destroy
     redirect_to provider_admin_dashboard_path
   end
 
-  protected
+  private
+
+  def find_backend_api
+    current_account.backend_apis.find(params[:service][:backend_api])
```

Comment:
`current_account.backend_apis.find(params.require(:service).require(:backend_api])` because otherwise if `service` is not in the params, it will raise `NoMethodError`